index.scss

```diff
@@ -6,66 +6,72 @@
 }
 
 .video-js {
+  $vjs-font-size: 10;
+  $font-size: 12;
+  $shelf-width: 25%;
+
+  $modal-padding-tb: 20px;
+  $modal-padding-lr: 24px;
+
   .vjs-dock-text,
   .vjs-dock-shelf {
+    font-family: "Helvetica Neue", Helvetica, Arial, sans-serif;
+    font-size: #{$font-size / $vjs-font-size}em; // eq. 12px
+    height: 50%;
     pointer-events: none;
+    position: absolute;
+    top: 0;
   }
 
   .vjs-dock-text {
+    left: 0;
     margin: 0;
     width: 100%;
-    height: 30%;
-    float: left;
-    padding: 0 10% 0 1em;
-    position: relative;
-  }
 
-  .vjs-dock-title,
-  .vjs-dock-description {
-    overflow: hidden;
-    text-overflow: ellipsis;
-    white-space: nowrap;
+    // Adopt some core styles from .vjs-modal-dialog from video.js
+    background: rgba(0, 0, 0, 0.8);
+    background: -webkit-linear-gradient(-90deg, rgba(0, 0, 0, 0.8) 25%, rgba(255, 255, 255, 0));
+    background: linear-gradient(180deg, rgba(0, 0, 0, 0.8) 25%, rgba(255, 255, 255, 0));
```

Comment:
I just realized that this looks terrible on Safari. Do you think you'll have a chance to take a look at that?